main/templates/partials/notifications.html

```diff
@@ -0,0 +1,8 @@
+<div class="notifications site-wide d-none">
+  {% if notification %}
+    <div class="notification" data-notification-id="{{ notification.id }}">
+      <a href="#" class="close-notification"></a>
```

Comment:
We can get rid of this element. We'll only be using this element to extract the HTML and show it in the React component, which will have its own close button

Reply:
ok